pydarn/plotting/acf.py

```diff
@@ -12,7 +12,7 @@
 from datetime import datetime
```

Comment:
Are these changes from another PR?

Reply:
Nope I found a bug in the code lol

Reply:
Yeah, I realized when you said when we chatted before! I didn't see anything obvious on code review for thsi so fine with me.